src/cdk/stack.ts

```diff
@@ -31,6 +31,16 @@ export class MiraServiceStack extends cdk.Stack {
           account: account.env.account
         }
       })
+
+      Tag.add(this, 'StackName', this.stackName)
+      Tag.add(this, 'CreatedBy', account.env.account)
```

Comment:
`account` will point to the current account number right? what is expected value here?

Reply:
That's right, isn't that the owner of the stack?

Reply:
Hmm, the account is where the stack is deployed, so we already know that information when looking at stacks.
The value of this field should be the userid of the user who created the stack.